examples/incorrect/assignment/descriptions.json

```diff
@@ -0,0 +1,34 @@
+{
+  "examples": [
+    {
+      "path": "assignment-incorrect-types.cac",
+      "errors": {
+        "lexerErrors": false
+      }
+    },
+    {
+      "path": "descriptions.json",
```

Comment:
I guess this file shouldn't contain it's own description?

Reply:
Nice catch, I didn't even notice, will fix